src/tables/cff.js

```diff
@@ -919,7 +919,8 @@ function makeStringIndex(strings) {
     ]);
     t.strings = [];
     for (var i = 0; i < strings.length; i += 1) {
-        t.strings.push({name: 'string_' + i, type: 'STRING', value: strings[i]});
+        if (strings[i])
+            t.strings.push({name: 'string_' + i, type: 'STRING', value: strings[i]});
     }
```

Comment:
I agree. The string index is a collection of all strings used in the font. If we used an undefined string somewhere, we should catch it *before* we try to create the index. 

@felipesanches do you have any example code that triggers this issue?